docs/contributing.md

```diff
@@ -80,6 +80,16 @@ For example, to run a single test script:
 $ scripts/test tests/test_cli.py
 ```
 
+Some of the tests start separate subprocesses. These tests are more complex in some ways, and can take longer, than the standard single-process tests. A [pytest mark](https://docs.pytest.org/en/latest/example/markers.html) is included to help control the behavior of subprocess tests.
+
+To run the test suite without subprocess tests:
+
+```shell
+$ scripts/test -m "not subprocess"
```

Comment:
Is it possible to not test by default, and do the opposite: `scripts/test -m "subprocess"`?

Reply:
We can. The way to disable subprocess tests by default is to add the `-m "not subprocess"` to `addopts` in `pyproject.toml`.

The downsides to this approach are:

- When we _do_ want to run the subprocess tests, test commands and invocation become more complicated. We would have to add another marker called `unmarked` and add it to all the other tests (https://github.com/pytest-dev/pytest/discussions/8573), then do `-m "subprocess or unmarked"` to run all the tests. I tried it but pytest is still skipping a lot of tests and I'm having trouble with it.
- Coverage reports will fail by default, which may confuse people.
- It doesn't match up as well with the [pytest docs on custom markers](https://docs.pytest.org/en/latest/example/markers.html) (the docs recommend the `"not subprocess"` approach for skipping tests with custom markers).

I prefer the `"not subprocess"` approach overall.

Reply:
I'm inclined on removing the `workers.py` module from uvicorn.